src/Illuminate/Console/Scheduling/Event.php

```diff
@@ -159,6 +159,21 @@ protected function callAfterCallbacks(Container $container)
 		}
 	}
 
+    /**
+     * Test if PHP is running on Windows.
+     * See http://stackoverflow.com/questions/5879043/php-script-detect-whether-running-under-linux-or-windows
+     *
+     * @return bool
+     */
+    public function isPhpOnWindowsServer()
+    {
+        if (strncasecmp(PHP_OS, 'WIN', 3) == 0) {
+            return true;
+        }
+
+        return false;
```

Comment:
You can turn this into a one-liner like this:

    return strncasecmp(PHP_OS, 'WIN', 3) == 0;